ripe/atlas/tools/renderers/traceroute_aspath.py

```diff
@@ -0,0 +1,78 @@
+import json
+import urllib2
+
+from ..ipcache import ip_cache
+from .base import Renderer as BaseRenderer
+from .base import Result
+
+
+class Renderer(BaseRenderer):
+
+    RENDERS = [BaseRenderer.TYPE_TRACEROUTE]
+
+    def __init__(self):
+        self.paths = {}
+
+        # Number of different ASs starting from the end of the traceroute path.
+        #
+        #TODO: if a method to pass options to renderers will be defined
+        # this could be set by user input.
+        self.RADIUS = 2
+        
+    @staticmethod
+    def _get_asns_for_output(asns,radius):
+        asns_with_padding = [""]*radius + asns
```

Comment:
Nitpick.  pep8 requires spaces on either side of that `*`.

Don't worry, I think this is my last comment :-)